src/app/drive/components/DriveExplorer/DriveExplorer.tsx

```diff
@@ -665,7 +671,7 @@ const DriveExplorer = (props: DriveExplorerProps): JSX.Element => {
       <EditFolderNameDialog />
       <UploadItemsFailsDialog />
       <MenuItemToGetSize />
-      <BannerWrapper />
+      {envService.isProduction() && <BannerWrapper />}
```

Comment:
Why display the banner only in production?

Reply:
Because for testing or development it bothers, at least for me. But I can add it again